Bio-KBase-Auth/lib/Bio/KBase/AuthToken.pm

```diff
@@ -585,6 +592,54 @@ sub go_request {
 
 }
 
+sub _get_token {
+    my $self = shift @_;
+    my %p = @_;
+
+    my $json;
+    eval {
+	my $baseurl = $Bio::KBase::Auth::AuthSvcHost;
```

Comment:
The url should be passed in to the new() method rather than from the config. I assume you know, but just checking.